FastSurferCNN/data_loader/conform.py

```diff
@@ -95,75 +93,91 @@ def make_parser() -> argparse.ArgumentParser:
         dest="check_only",
         default=False,
         action="store_true",
-        help="If True, only checks if the input image is conformed, and does not "
-             "return an output.",
+        help="If True, only checks if the input image is conformed, and does not return an output.",
     )
     parser.add_argument(
         "--seg_input",
         dest="seg_input",
-        default=False,
         action="store_true",
-        help="Specifies whether the input is a seg image. If true, the check for "
-             "conformance disregards the uint8 dtype criteria. Use --dtype any for "
-             "equivalent results. --seg_input overwrites --dtype arguments.",
+        help="Specifies whether the input is a seg image. If true, the default values for dtype and rescale are "
+             "changed to 'integer' and 'none', which only means the dtype must be an integer and no rescaling is "
+             "performed.",
     )
     parser.add_argument(
         "--vox_size",
         dest="vox_size",
         default=1.0,
         type=__vox_size,
-        help="Specifies the target voxel size to conform to. Also allows 'min' for "
-             "conforming to the minimum voxel size, otherwise similar to mri_convert's "
-             "--conform_size <size> (default: 1, conform to 1mm).",
+        help="Specifies the target voxel size to conform to (deactivated by 'any'). Also allows 'min' for conforming "
+             "to the minimum voxel size, otherwise similar to mri_convert's --conform_size <size> (default: 1, conform "
+             "to 1mm).",
     )
     parser.add_argument(
         "--conform_min",
-        dest="conform_min",
-        default=False,
-        action="store_true",
-        help="Specifies whether the input is or should be conformed to the "
-        "minimal voxel size (used for high-res processing) - overwrites --vox_size.",
+        dest="vox_size",
+        action="store_const",
+        const="min",
+        help="Specifies whether the input is or should be conformed to the minimal voxel size (used for high-res "
+             "processing) - overwrites --vox_size.",
+    )
+    parser.add_argument(
+        "--img_size",
+        dest="img_size",
+        default="auto",
+        type=__img_size,
+        help="Specifies the image size to conform to (deactivated by 'any'). Also allows 'fov' for conforming to the "
+             "field of view.",
+    )
+    parser.add_argument(
+        "--rescale",
+        dest="rescale",
+        default="auto",
+        choices=("auto", "robust", "none"),
+        help="Specifies whether image intensities should be rescaled, 'robust' will robustly intensities to the output "
```

Comment:
I don't understand the default here. from the description it seems that auto is the same as robust?

Reply:
OK, becomes clear later. It should say in the description that robust depends on the --dtype flag, while auto always overwrites --dtype to uchar? or what?

Reply:
I should word this better.

"auto" (default): depends on the `--dtype flag",  if the target dtype is uint8 (the default), => like "robust" else "none"
"robust": robustly rescale intensities to 0-maxvalue of dtype (e.g. 0-255 for uint8)
"none": disabled/no rescaling